Tasks/google-play-release/task.loc.json

```diff
@@ -14,7 +14,7 @@
   ],
   "version": {
     "Major": "3",
-    "Minor": "174",
+    "Minor": "182",
```

Comment:
Please bump version to current sprint number

Reply:
Thanks! Fixed